app/models/form_submission.rb

```diff
@@ -10,7 +10,47 @@ class FormSubmission < ApplicationRecord
 
   validates :form_type, presence: true
 
-  def latest_pending_attempt
-    form_submission_attempts.where(aasm_state: 'pending').order(created_at: :asc).last
+  class << self
+    # rubocop:disable Metrics/MethodLength
+    def with_latest_intake(user_account)
+      # This query retrieves distinct form submissions for a given user account,
+      # along with the latest benefits intake UUID for each form. If a form has any
+      # submission attempts, it uses the latest attempt's UUID; otherwise, it falls
+      # back to the form submission's original UUID.
```

Comment:
Might it simplify the query if we give `FormSubmissionAttempt` a `benefits_intake_uuid` method that returns its own `benefits_intake_uuid || form_submission.benefits_intake_uuid`? (ie, conditionally delegate to the parent Form Submission)

Reply:
Agreed.  thanks!